src/v2/Apps/Show/Components/ShowNavigationBanner.tsx

```diff
@@ -0,0 +1,50 @@
+import { BoxProps } from "@artsy/palette"
+import React from "react"
+import { BackLink } from "v2/Components/Links/BackLink"
+import { ShowNavigationBanner_show } from "v2/__generated__/ShowNavigationBanner_show.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface ShowNavigationBannerProps {
+  show: ShowNavigationBanner_show
+}
+
+const ShowNavigationBanner: React.FC<ShowNavigationBannerProps & BoxProps> = ({
+  show,
+  ...rest
+}) => {
+  const { name, fair, isFairBooth, partner } = show
+
+  if (!fair?.href) {
+    return null
+  }
+
+  return (
+    <BackLink {...rest} to={`${fair.href}/exhibitors`}>
+      Back to {name}
+      {!isFairBooth && partner?.name && `at ${partner.name}`}
```

Comment:
**suggestion:** This implementation matches the [design spec](https://www.figma.com/file/aM3b5sodn04vMTigbQXACi/Fairs-2021?node-id=3474%3A17827). However it seems odd that a user who is on the "MOVART at African Galleries Now 2021" show page is being asked to go "Back to MOVART at African Galleries Now 2021."

I feel like the text should be "Back to African Galleries Now 2021" instead, and the show name should be omitted from this back link.